packages/shared/src/lib/core/nfts/utils/buildPersistedNftFromNftMetadata.ts

```diff
@@ -0,0 +1,14 @@
+import { NftStandard } from '../enums'
+import { INftInstance, IPersistedNftMetadata } from '../interfaces'
+import { NftMetadata } from '../types'
+
+export function buildPersistedNftFromNftMetadata(
+    metadata: NftMetadata,
+    instances?: INftInstance[]
+): IPersistedNftMetadata {
+    return {
+        standard: metadata.standard as NftStandard.Irc27,
```

Comment:
is this meant to erc721? either way confusing because the function name is generic